---
layout: archive
permalink: /
title: ""
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

Personal information
======
* Beijing Institutes of Life Sciences
* Chinese Academy of Sciences
* No.1 Beichen West Road
* Chaoyang District, Beijing, 100101, P.R. China

Education
======
* 09/2011-06/2015  <b>B.S.</b>, Life Sciences
  * School of Life Sciences
  * Peking University, P.R. China
* 09/2015-06/2021  <b>Ph.D.</b>, Genetics (with honors),
  * Beijing Institutes of Life Sciences (College of Life Sciences)
  * Chinese Academy of Sciences, P.R. China
  * Supervisor: Prof. Fangqing Zhao
  * Dissertion: Accurate quantification and reconstruction of circular RNAs using high-throughput sequencing data

Work experience
======
* 09/2021-07/2023  <b>Post-Doctoral Associate</b>
  * Computational Genomics Lab, Beijing Institutes of Life Science
  * Chinese Academy of Sciences, P.R. China
  * Supervisor: Prof. Fangqing Zhao
  * Special Research Assistant funded projects, Chinese Academy of Sciences
  * China National Postdoctoral Program for Innovative Talents
* 07/2023-now      <b>Associate research fellow</b>
  * Computational Genomics Lab, Beijing Institutes of Life Science
  * Chinese Academy of Sciences, P.R. China

Awards & Honors
======
* 2022 Excellent Doctoral Thesis, Beijing
* 2022 Excellent Doctoral Thesis, Chinese Academy of Sciences
* 2021 CAS President Scholarship (Special Prize), Chinese Academy of Sciences
* 2021 Pacemaker to Merit Student, University of Chinese Academy of Sciences
* 2019 Zhuliyuehua Outstanding Doctoral Award, Chinese Academy of Sciences
* 2019 Merit Student, University of Chinese Academy of Sciences
* 2017 Merit Student, University of Chinese Academy of Sciences

Publications
======

**Web of Science**: [AFS-9879-2022](https://www.webofscience.com/wos/author/record/AFS-9879-2022)

(# indicates co first authors, *corresponding authors)

15.	Vromman, M.#, Anckaert, J., Bortoluzzi, S, Buratin, A., Chen, C., Chu, Q., Chuang, T., Dehghannasiri, R., Dieterich, C., Dong, X., Flicek, P., Gaffo, E., Gu, W., He, C., Hoffmann, S., Izuogu, O., Jackson, M., Jakobi, T., Lai, E., Nuytens, J., Salzman, J., Santibanez-Koref, M., Stadler, P., Thas, O., Eynde, E., Verniers, K., Wen, G., Westholm, J., Yang, L., Ye, C., Yigit, N., Yuan, G., **Zhang, J.**, Zhao, F., Volders, P.*, Vandesompele, J.* (2023) Large-scale benchmarking of circRNA detection tools reveals large differences in sensitivity but not in precision. ***Nature Methods*** (**IF=48.0**) [[DOI]](https://doi.org/10.1038/s41592-023-01944-6) [[Shareable link]](https://rdcu.be/dgL3h)
14.	Hou, L.#, **Zhang, J.#*** & Zhao, F.* (2023). Full-length circular RNA profiling by nanopore sequencing with CIRI-long. ***Nature protocol*** (**IF=14.8**) [[DOI]](https://doi.org/10.1038/s41596-023-00815-w) [[Shareable link]](https://rdcu.be/c9KAb)
13.	Deshpande, D.#, Chhugani, K.#, Chang, Y., Karlsberg, A., Loeffler, C., **Zhang, J.**, Muszynska, A., Rotman, J., Tao, L., Martin, L. S., Balliu, B., Tseng, E., Eskin, E., Zhao, F., Mohammadi, P., Labaj, P. P., & Mangul, S. (2023). RNA-seq data science: From raw data to effective interpretation. ***Frontiers in Genetics*** (**IF=3.7**) [[DOI]](https://doi.org/10.3389/fgene.2023.997383)
12.	Wu. W.#, **Zhang, J.#**, & Zhao, F. (2022). Exploring the cellular landscape of circular RNAs using full-length single-cell RNA sequencing. ***Nature Communications***, 13, 3242. (**IF=16.6**) [[DOI]](https://doi.org/10.1038/s41467-022-30963-8)
11.	Yang, J.#, Hou, L.#, Wang, J.#, Xiao, L., **Zhang, J.**, Yin, N., Wei, Y., Yao, Su., Cheng, K., Zhang, W., Shi, Z., Wang, J., Jiang, H., Huang, N., You, Y., Lin, M., Shang, R., Zhao, Y.* & Zhao, F.* (2022). Unfavourable intrauterine environment contributes to abnormal gut microbiome and metabolome in twins. ***Gut***, 71:2451-2462. (**IF=24.5**) [[DOI]](https://doi.org/10.1136/gutjnl-2021-326482)
10.	Chen, S.#, Cao, X.#, **Zhang, J.#**, Wu, W., Zhang, B. & Zhao, F. (2022). circVAMP3 drives CAPRIN1 phase separation and inhibits hepatocellular carcinoma by suppressing c-Myc translation. ***Advanced Science***, 9(8): e2103817. (**IF=15.1**) [[DOI]](https://doi.org/10.1002/advs.202103817)
11. Chen, S.#, **Zhang, J.#**, & Zhao, F. (2022). Screening linear and circular RNA transcripts from stress granules. ***Genomics, Proteomics & Bioinformatics***, (**IF=6.409**) [[DOI]](https://doi.org/10.1016/j.gpb.2022.01.003)
12. **Zhang, J.** & Zhao, F. (2021). Characterizing circular RNAs using nanopore sequencing. ***Trends in Biochemical Sciences***. 46, 785–786 (**IF=13.8**) [[DOI]](https://doi.org/10.1016/j.tibs.2021.06.002)
13. **Zhang, J.** & Zhao, F. (2021). Reconstruction of circular RNAs using Illumina and Nanopore RNA-seq datasets. ***Methods*** 196, 17–22 (**IF=4.8**) [[DOI]](https://doi.org/10.1016/j.ymeth.2021.03.017)
14. **Zhang, J.#**, Hou, L.#, Zuo, Z.#, Ji, P., Zhang, X., Xue, Y. & Zhao, F. (2021). Comprehensive profiling of circular RNAs with nanopore sequencing and CIRI-long. ***Nature Biotechnology***, 39, 836–845. (**IF=46.9**) [[DOI]](https://doi.org/10.1038/s41587-021-00842-6) [[Nature Biotechnology]](https://www.nature.com/articles/s41587-021-00916-5) [[2021 Top 10 Advances in Bioinformatics in China]](http://gpb.big.ac.cn/news/1015)
15. Li, X.#, Zheng, J.#, Ma, X.#, Zhang, B., **Zhang, J.**, Wang, W., Sun, C., Wang, Y., Zheng, J., Chen, H., Tao, J., Wang, H., Zhang, F., Wang, J., & Zhang, H. (2021). The Oral Microbiome of Pregnant Women Facilitates Gestational Diabetes Discrimination. ***Journal of Genetics and Genomics***, 48, 32–39 (**IF=5.9**) [[DOI]](https://doi.org/10.1016/j.jgg.2020.11.006)
16. **Zhang, J.**, Chen, S., Yang, J., & Zhao, F. (2020). Accurate quantification of circular RNAs identifies extensive circular isoform switching events. ***Nature Communications***, 11(1), 90. (**IF=16.6**) [[DOI]](https://doi.org/10.1038/s41467-019-13840-9) [[2020 Top 10 Advances in Bioinformatics in China]](http://gpb.big.ac.cn/news/1014)
17. Ji, P.#, Wu, W.#, Chen, S.#, Zheng, Y., Zhou, L., **Zhang, J.**, Cheng, H., Yan, J., Zhang, S., Yang, P., & Zhao, F. (2019). Expanded Expression Landscape and Prioritization of Circular RNAs in Mammals. ***Cell Reports***, 26(12), 3444–3460.e5. (**IF=8.8**) [[DOI]](https://doi.org/10.1016/j.celrep.2019.02.078)
18. Gao, Y., **Zhang, J.**, & Zhao, F. (2018). Circular RNA identification based on multiple seed matching. ***Briefings in Bioinformatics***, 19(5), 803–810. (**IF=9.5**) [[DOI]](https://doi.org/10.1093/bib/bbx014)
19. Gao, Y.#, Wang, J.#, Zheng, Y.#, **Zhang, J.**, Chen, S., & Zhao, F. (2016). Comprehensive identification of internal structure and alternative splicing events in circular RNAs. ***Nature Communications***, 7, 12060. (**IF=16.6**) [[DOI]](https://doi.org/10.1038/ncomms12060)

Invited Talks & Conference Presentations
======
* 2021 The 11th youth forum of national symposium of Chinese RNA society. July 2-5th. Dalian, China (Invited talk)
* 2019 The 1st National Postgraduate Forum of Basic Medicine held by the Savaid Medical School, University of Chinese Academy of Sciences. November 30th. Beijing, China (Invited talk)

Journal Reviewer
======

Reviewers of ***Briefings in Bioinformatics***, ***Genomics Proteomics and Bioinformatics***, ***mLife***